samples/videoDecodeRGB/videodecrgb.cpp

```diff
@@ -43,11 +43,8 @@ THE SOFTWARE.
 #include "roc_video_dec.h"
 #include "colorspace_kernels.h"
 #include "resize_kernels.h"
+#include "video_post_process.h"
```

Comment:
Please remove the `#include "resize_kernels.h"` and `#include "resize_kernels.h"` from lines 44 and 45 of this file, since `#include "video_post_process.h"` already includes them.

Reply:
done